fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/Sum.java

```diff
@@ -49,6 +50,9 @@ public DataType getDataType() {
         DataType dataType = child().getDataType();
         if (dataType instanceof LargeIntType) {
             return dataType;
+        } else if (dataType instanceof DecimalType) {
```

Comment:
Why original code returns Double Type? It seems that it throws Exception.

Reply:
DecimalType is a SubClass of FractionalType